src/pages/en/guides/deploy/github.md

```diff
@@ -15,15 +15,27 @@ Astro maintains the official `withastro/action` to deploy your project with very
 
 1. Set the [`site`](/en/reference/configuration-reference/#site) and, if needed, [`base`](/en/reference/configuration-reference/#base) options in `astro.config.mjs`.
     - `site` should be something like `https://<YOUR_USERNAME>.github.io`
-    - `base` should be your repository’s name starting with a forward slash, for example `/my-repo`.
+    - `base` should be your repository’s name starting with a forward slash, for example `/my-repo`. This is so that Astro understands your website's root is `/my-repo`, rather than the default `/`.
     
```

Comment:
Is it too much to show an `astro.config.mjs` example here with those fields filled out?

Reply:
@odwrotnie was asking for an example in #1376. They managed to [misplace the parameters](https://github.com/instap-app/website-definition/blob/4fd94706505c385128284a3f0b27eb85ff2d85ca/astro.config.mjs#L9-L10) outside of the defineConfig object, because they didn't find this clear enough.